sql-native/src/main/java/org/ballerinalang/sql/nativeimpl/ExecuteProcessor.java

```diff
@@ -46,16 +49,19 @@
 import static org.ballerinalang.sql.utils.Utils.closeResources;
 import static org.ballerinalang.sql.utils.Utils.getGeneratedKeys;
 import static org.ballerinalang.sql.utils.Utils.getSqlQuery;
-import static org.ballerinalang.sql.utils.Utils.setParams;
+
 
 /**
- * This class holds the utility methods involved with executing the query which does not return rows.
+ * This class contains methods for executing SQL queries.
  *
- * @since 1.2.0
+ * @since 0.5.6
  */
-public class ExecuteUtils {
-
+public class ExecuteProcessor {
     public static Object nativeExecute(BObject client, Object paramSQLString) {
+        return nativeExecute(client, paramSQLString, StatementParameterProcessor.getInstance());
+    }
+    public static Object nativeExecute(BObject client, Object paramSQLString,
```

Comment:
We don't need a separate method and this logic can be moved to the calling method itself.

Reply:
+1

Reply:
Resolved with [`666f5cm`](https://github.com/ballerina-platform/module-ballerina-sql/pull/104/commits/666f5c49ee9e3cb4dc0575a792bc643ee9523451)